packages/apidom-ls/src/config/openapi/openapi3_1/documentation.ts

```diff
@@ -0,0 +1,20 @@
+const documentation = [
```

Comment:
Can we enumarate here in comment, what fields are missing?

It is very important not to produce incomplete rules for specification object. If somebody comes after you, the person just does't know if the rules are complete or not and always have to check. The rules of thumb is that when `documentation.ts` file is produced, it is expected to be always complete.

Reply:
Done:
- Added fields that should render
- Make a comment where field is expected.

All fixed fields referenced [here](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#fixed-fields) is now listed or commented in `openapi3_1/documentation.ts`